packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec

```
@@ -45,8 +50,13 @@ install -Dm644 dashboards/provider-sleha.yaml %{buildroot}%{provisioning_dir}/pr
 %defattr(-,root,root)
 %doc dashboards/README.md
 %license LICENSE
-%attr(0755,grafana,grafana) %dir %{dashboards_dir}/sleha
+%attr(0755,root,root) %dir %{dashboards_dir}/sleha
 %attr(0644,grafana,grafana) %config %{dashboards_dir}/sleha/*
-%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
+%attr(0644,grafana,grafana) %config %{provisioning_dir}/provider-sleha.yaml
+%attr(0644,grafana,grafana) %dir  %{_sysconfdir}/grafana
+%attr(0644,grafana,grafana) %dir  %{_sysconfdir}/grafana/provisioning
+%attr(0644,grafana,grafana) %dir  %{_sysconfdir}/grafana/provisioning/dashboards
+%attr(0644,grafana,grafana) %dir  %{_localstatedir}/lib/grafana
+%attr(0644,grafana,grafana) %dir  %{_localstatedir}/lib/grafana/dashboards
```

these have the correct user and group, but they are still directories:
```suggestion
%attr(0755,grafana,grafana) %dir  %{_localstatedir}/lib/grafana
%attr(0755,grafana,grafana) %dir  %{_localstatedir}/lib/grafana/dashboards
```